mlir/include/mlir/Target/LLVMIR/ModuleImport.h

```diff
@@ -350,7 +355,8 @@ class ModuleImport {
   DictionaryAttr convertParameterAttribute(llvm::AttributeSet llvmParamAttrs,
                                            OpBuilder &builder);
   /// Converts the parameter and result attributes attached to `call` and adds
-  /// them to the `callOp`.
+  /// them to the `callOp`. Implemented in terms of the other definition of
+  /// the public definition of convertParameterAttributes.
   void convertParameterAttributes(llvm::CallBase *call, CallOpInterface callOp,
```

Comment:
I guess this function would be a good use case for a ArgumentAndResultAttributeInterface.

Reply:
Do you mean In the future or as part of this PR?

Reply:
No this is definitely a separate PR.